ribasim_qgis/widgets/dataset_widget.py

```diff
@@ -50,10 +50,10 @@ def __init__(self, parent: QWidget | None):
         self.setSizePolicy(QSizePolicy.Minimum, QSizePolicy.Preferred)
         self.setHeaderLabels([""])
```

Comment:
Does it make sense to write `"  Layer"` here as a label here? (Note the spaces in front, otherwise it looks too cramped...)

![image](https://github.com/Deltares/Ribasim/assets/13662783/ec7cabeb-8c38-46b3-92ba-3305a196ce54)